docs/pages/admin-guides/management/operations/backup-restore.mdx

```diff
@@ -2,211 +2,118 @@
 | DynamoDB | [Follow AWS's guidelines for backup and restore](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) |
 | etcd | [Follow etcd's guidelines for disaster recovery](https://etcd.io/docs/v2/admin_guide) |
 | Firestore | [Follow GCP's guidelines for automated backups](https://firebase.google.com/docs/database/backups) |
+| Azure Database for PostgreSQL | [Follow Azure's guidelines for automated backups](https://learn.microsoft.com/en-us/azure/backup/backup-azure-database-postgresql) |
 
-</TabItem>
-<TabItem label="Cloud-Hosted">
-
-Teleport Enterprise Cloud manages all Auth Service and Proxy Service backups.
-
-While Teleport Nodes are stateless, you should ensure that you can restore their
-configuration files.
-
-</TabItem>
-</Tabs>
-
-### Teleport resources
-
-Teleport uses YAML resources for roles, Trusted Clusters, local users, and authentication connectors.
-These could be created via `tctl` or the Web UI.
-
-You should back up your dynamic resource configurations to ensure that you can restore them in case of an outage.
-
-## Our recommended backup practice
-
-If you're running Teleport at scale, your teams need to have an automated way to restore Teleport. At a high level, this is our recommended approach:
-
-<Tabs>
-<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
-
-- Persist and back up your backend.
-- Share that backend among Auth Service instances.
-- Store your dynamic resource configurations as discrete files in a git
-  repository.
-- Have your continuous integration system run `tctl create -f *.yaml` from the
-  git repository. The `-f` flag instructs `tctl create` not to return an error
-  if a resource already exists, so this command can be run regularly.
-
-</TabItem>
-<TabItem scope={["cloud","team"]} label="Cloud-Hosted">
-
-- Store your dynamic resource configurations as discrete files in a git
-  repository.
-- Have your continuous integration system run `tctl create -f *.yaml` from the
-  git repository. The `-f` flag instructs `tctl create` not to return an error
-  if a resource already exists, so this command can be run regularly.
-
-</TabItem>
-</Tabs>
-
-## Migrating backends
-<Tabs>
-<TabItem scope={["enterprise"]} label="Teleport Enterprise">
-
-You can export a collection of resources from
-Teleport using the below command. This feature helps for migrating
-from one backend to another.
-
-Using `tctl get all --with-secrets` will retrieve the below items:
+## Dynamic resources
 
-- Users
-- Certificate Authorities
-- Trusted Clusters
-- Connectors:
-  - GitHub
-  - SAML
-  - OIDC
-- Roles
+Teleport uses [dynamic
+resources](../../infrastructure-as-code/infrastructure-as-code.mdx) for roles,
+local users, authentication connectors, and other configurations. We recommend
+that you back up your configuration resources because they are essential to the
+normal operation of your cluster.
```

Comment:
But this information is backed up if you back up the cluster state backend. Should we make that more clear?